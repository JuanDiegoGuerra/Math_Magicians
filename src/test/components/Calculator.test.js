import { render } from '@testing-library/react';
import Calculator from '../../components/calculator';

describe('Calculator', () => {
  it('Renders calculator correctly', () => {
    const { container } = render(<Calculator />);
    expect(container).toMatchSnapshot();
  });
});
