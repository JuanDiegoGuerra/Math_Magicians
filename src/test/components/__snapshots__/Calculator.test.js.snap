// Jest Snapshot v1, https://goo.gl/fbAQLP

exports[`Calculator Renders calculator correctly 1`] = `
<div>
  <div
    class="grid-container"
  >
    <div
      class="calc-number calc-screen"
      data-testid="screen-element"
    >
       
      0
       
    </div>
    <div
      class="calc-number"
      role="button"
      tabindex="0"
    >
      AC
    </div>
    <div
      class="calc-number"
      role="button"
      tabindex="0"
    >
      +/-
    </div>
    <div
      class="calc-number"
      role="button"
      tabindex="0"
    >
      %
    </div>
    <div
      class="calc-number calc-op"
      role="button"
      tabindex="0"
    >
      รท
    </div>
    <div
      class="calc-number calc-7"
      role="button"
      tabindex="0"
    >
      7
    </div>
    <div
      class="calc-number calc-8"
      role="button"
      tabindex="0"
    >
      8
    </div>
    <div
      class="calc-number calc-9"
      role="button"
      tabindex="0"
    >
      9
    </div>
    <div
      class="calc-number calc-op"
      role="button"
      tabindex="0"
    >
      x
    </div>
    <div
      class="calc-number calc-4"
      role="button"
      tabindex="0"
    >
      4
    </div>
    <div
      class="calc-number calc-5"
      role="button"
      tabindex="0"
    >
      5
    </div>
    <div
      class="calc-number calc-6"
      role="button"
      tabindex="0"
    >
      6
    </div>
    <div
      class="calc-number calc-op"
      role="button"
      tabindex="0"
    >
      -
    </div>
    <div
      class="calc-number calc-1"
      role="button"
      tabindex="0"
    >
      1
    </div>
    <div
      class="calc-number calc-2"
      role="button"
      tabindex="0"
    >
      2
    </div>
    <div
      class="calc-number calc-3"
      role="button"
      tabindex="0"
    >
      3
    </div>
    <div
      class="calc-number calc-op"
      role="button"
      tabindex="0"
    >
      +
    </div>
    <div
      class="calc-number calc-0"
      role="button"
      tabindex="0"
    >
      0
    </div>
    <div
      class="calc-number"
      role="button"
      tabindex="0"
    >
      .
    </div>
    <div
      class="calc-number calc-op"
      role="button"
      tabindex="0"
    >
      =
    </div>
  </div>
</div>
`;
